.travis.yml

```diff
@@ -18,8 +6,22 @@ before_install:
   - (test -x $HOME/.cargo/bin/mdbook-linkcheck || cargo install mdbook-linkcheck)
   - cargo install-update -a
 
+language: rust
```

Comment:
I think you need to put this matrix under the test-features stage or it will only apply to the test stage. And please make sure that the deploy stage is only run on a single variant (nightly, preferably)

Reply:
Thanks. Struggling with their documentation...